### DISCUSSION TITLE
Sending XCH to my Chia Wallet

### DISCUSSION THREAD

**Discussion by camtrain** [Chia-Network/chia-blockchain#5128] [Support]
I transferred by chia from LBank to my Chia Wallet this morning, used the correct receive address on my chia wallet, and got a confirmation email from LBank stating the withdrawal was successful. It has been about 6 hours since I got the confirmation email however and I still have not had any update in my chia wallet balance. Any help here?

**Comment by cccat6** [comment]
Did your wallet get synced? Check www.chiaexplorer.com and see if the transaction succeed.

**Reply by camtrain** [reply]
Thank you - this was helpful. The transaction went through but I am not synced. How do I sync?

**Reply by cccat6** [reply]
If your wallet consistently "not synced". Maybe delete the wallet db file and let it resync. If it shows "syncing", just wait for it.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
